browser/brave_ads/tooltips/ads_captcha_tooltip.cc

```diff
@@ -0,0 +1,41 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/brave_ads/tooltips/ads_captcha_tooltip.h"
+
+#include "brave/browser/brave_ads/ads_service_factory.h"
```

Comment:
you're missing an include (and also dep) for AdsService

Reply:
actually see comment below, you should have the captcha service factory and captcha service headers here